### DISCUSSION TITLE
Terminal Window stays open?

### DISCUSSION THREAD

**Discussion by keygenesis** [ClassicOldSong/Apollo#85] [General]
Hello, with Sunshine I was able to have it launch on start up and just keep it open in the taskbar, but Apollo opens a command prompt that stays open, and when I close it it closes the whole program (I'm on Windows). Is this intentional, or is there a way to have a hidden command interface? Thanks!

**Comment by ClassicOldSong** [comment]
You might be using it wrong. Apollo launched from shortcut created by the installer should not have kept a terminal window on the foreground, maybe you're running the exe directly?

**Reply by keygenesis** [reply]
Ah yes I have been running the exe directly, but when I run it from the shortcut a terminal window opens that says

`[server_cmd] -- [[{"name":"Bubbles","cmd":"bubbles.scr","elevated":false}]]
[2024-10-28 12:24:32.918300] [0x000010d0] [info]    OpenService() failed: 1060
[server_cmd] -- [[{"name":"Bubbles","cmd":"bubbles.scr","elevated":false}]]
[2024-10-28 12:24:32.968974] [0x00005620] [info]    OpenService() failed: 1060
Starting Sunshine...Waiting for Web UI to be ready....`

and nothing happens, not sure why.

**Reply by ClassicOldSong** [reply]
Do you having Administrator permission? It seems that either you have an incomplete installation or lacking permission.
